lib/orator/index.js

```diff
@@ -327,18 +327,29 @@ class Orator {
           avatarURL
         } = response
 
-        const hook = await findOrCreateManagedWebhook(
-          msg.channel,
-          bot.user.id,
-          this.webhookName
-        )
+        try {
```

Comment:
It might look better if you returned the `findOrCreate` with the createMessage chained on with a `then`; Then you'd be able to just have a single catch on the chain that handles if the error is a permission error